components/longSparkArrow/LongSparkArrow.styles.js

```diff
@@ -0,0 +1,107 @@
+import { P } from '../../style/typography';
+import styled from 'styled-components';
+import { colors } from '../../style/colors';
+import { fontWeights } from '../../style/typography';
+import { min } from '../../lib/responsive';
+
+const UnfilledSparkArrow = styled.img`
+  position: absolute;
+  transform: rotate(90deg);
+  z-index: 1;
+  @media ${min.mobile} {
+    top: -1.3em;
+    right: 12em;
+    height: 30em;
+  }
+  @media ${min.tablet} {
+    top: -1em;
+    right: 8.2em;
+    height: 21em;
+  }
+  @media ${min.desktop} {
+    top: 2.5em;
+    right: 5.7em;
+    height: 14em;
+  }
+`;
+
+const FilledSparkArrow = styled.img`
+  position: absolute;
+  transform: rotate(90deg);
+  z-index: 1;
+  @media ${min.desktop} {
+    top: 2.1em;
+    right: 8em;
+    height: 19em;
+  }
+`;
+
+const ArrowTextHover = styled(P)`
+  position: absolute;
+  z-index: 2;
+  right: 0;
+  text-align: right;
+  color: ${colors.BROWN};
+  font-weight: ${fontWeights.bold};
+  @media ${min.desktop} {
+    padding-top: 9.8em;
+    width: 20em;
+  }
+`;
+
+const ArrowText = styled(P)`
+  position: absolute;
+  z-index: 2;
+  right: 0;
+  text-align: right;
+  color: ${colors.BROWN};
+  @media ${min.mobile} {
+    padding-top: 4.5em;
+    width: 20em;
+  }
+  @media ${min.tablet} {
+    padding-top: 4.2em;
+    width: 20em;
```

Comment:
you can remove `width: 20em` here and on line 68 since it will continue to apply from line 60